Zend/zend_max_execution_timer.h

```diff
@@ -19,12 +19,14 @@
 
 # ifdef ZEND_MAX_EXECUTION_TIMERS
 
+BEGIN_EXTERN_C()
 #include "zend_long.h"
```

Comment:
Please move the `#include` outside of the `BEGIN_EXTERN_C()` block

Reply:
done